environment.yml

```diff
@@ -1,4 +1,4 @@
-name: nf-core-coproid-1.0dev
+name: nf-core-coproid-1.0
 channels:
   - conda-forge
   - bioconda
```

Comment:
bioconda needs to have highest priority, then conda-forge.

Reply:
Changed in #7